index.js

```diff
@@ -78,6 +76,11 @@ class SQSConsumer extends EventEmitter {
 		}
 	}
 
+	getMaxNumberOfMessages() {
+		const max = this.batchSize - this.numActiveMessages;
+		return max > 10 ? 10 : max;
```

Comment:
Is it worth adding a guard for making sure the value is minimum 1? The docs state that 1 is the minimum value, so I'm guessing it'll throw an error if we send 0.

Not sure if we need to worry about this, it all depends on whether there's a chance of a job completing in-between the check for `this.numActiveMessages < this.batchSize` and the call to `getMaxNumberOfMessages`. It's not obvious to me if calling `await this.poll` means we go back to the event loop or not.

Reply:
Good call, added safe guard to at least return 1